src/App.jsx

```diff
@@ -102,6 +102,7 @@ class App extends Component {
               );
             }}
           />
+          <Route path="/login" component={Login} />
```

Comment:
remove this route. When logged in, users shouldn't be able to access the login page